Tests/SourceKitLSPTests/LocalSwiftTests.swift

```diff
@@ -1476,4 +1476,55 @@ final class LocalSwiftTests: XCTestCase {
     data = EditorPlaceholder(text)
     XCTAssertNil(data)
   }
+  
+  func testIncrementalParse() throws {
+    let url = URL(fileURLWithPath: "/\(UUID())/a.swift")
+    let uri = DocumentURI(url)
+
+    sk.allowUnexpectedNotification = false
+    
+    sk.send(DidOpenTextDocumentNotification(textDocument: TextDocumentItem(
+      uri: uri,
+      language: .swift,
+      version: 0,
+      text: """
+      func foo() {
+      }
+      class bar {
+      }
+      """
+    )))
+    
+    let didChangeTextDocumentExpectation = XCTestExpectation(description: "didChangeTextDocument")
```

Comment:
You can do `self.expectation` here. That way we will get an error if you don’t wait for the expectation

```suggestion
    let didChangeTextDocumentExpectation = self.expectation(description: "didChangeTextDocument")
```